src/plugins/intel_npu/src/plugin/npuw/weights_bank.hpp

```diff
@@ -10,26 +10,35 @@
 #include <unordered_map>
 
 #include "openvino/openvino.hpp"
+#include "openvino/runtime/iremote_context.hpp"
 #include "openvino/runtime/make_tensor.hpp"
 #include "openvino/runtime/tensor.hpp"
+#include "plugin.hpp"
 
 namespace ov {
 namespace npuw {
 namespace weights {
 
 class Bank {
 public:
-    Bank() = default;
+    Bank(const std::shared_ptr<const ov::IPlugin>& plugin) : m_plugin(plugin){};
```

Comment:
if it is really needed here, you've forgot `explicit` (my top-1 review comment on Github)

Reply:
Done